CODEOWNERS

```diff
@@ -1,2 +1,2 @@
 # generated by giantswarm/github actions - changes will be overwritten
-* @giantswarm/area-kaas
+* @giantswarm/team-cabbage
```

Comment:
As mentioned in the comment, this file is managed by our automation and you need to change it here https://github.com/giantswarm/github/blob/master/repositories/area-kaas.yaml#L17-L22